### PR TITLE
Add filter example

### PR DESCRIPTION
**Pull request by diego-vieira**
Save from trying to figure out the syntax accepted by the engine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,6 +228,15 @@ $index->search('prince', ['limit' => 1])
 }
 ```
 
+With limit and filter, both single and double quotes are supported.
+```php
+// Enclosing with double quotes
+$index->search('prince', ['limit' => 1, 'filter' => "title = 'Le Petit Prince' OR book_id = 456"]);
```
Comment (curquiza) [inline README.md:234]:
```suggestion
$index->search('prince', ['limit' => 1, 'filters' => "title = 'Le Petit Prince' OR book_id = 456"]);
```

File: README.md [hunk 2]
```diff
@@ -228,6 +228,15 @@ $index->search('prince', ['limit' => 1])
...
+
+// Enclosing with single quotes
+$index->search('prince', ['limit' => 1, 'filter' => 'title = "Le Petit Prince" OR book_id = 456']);
```
Comment (curquiza) [inline README.md:237]:
```suggestion
$index->search('prince', ['limit' => 1, 'filters' => 'title = "Le Petit Prince" OR book_id = 456']);
```

File: README.md (hunk 1 above)
Comment (diego-vieira) [reply README.md:234]: oops, not sure how that end up there lol - thanks

File: README.md [hunk 3]
```diff
@@ -228,6 +228,15 @@ $index->search('prince', ['limit' => 1])
 }
 ```
 
+With limit and filter, both single and double quotes are supported.
+```php
+// Enclosing with double quotes
+$index->search('prince', ['limit' => 1, 'filters' => "title = 'Le Petit Prince' OR book_id = 456"]);
+
+// Enclosing with single quotes
+$index->search('prince', ['limit' => 1, 'filters' => 'title = "Le Petit Prince" OR book_id = 456']);
```
[the remark this answers is not in the document]
Comment (curquiza) [reply README.md:237]: Oh good to know! I will update the README according to your remark then!

[the remark this answers is not in the document]
Comment (diego-vieira) [reply README.md:237]: @Kerollmops thanks, I have now updated as suggested. https://github.com/meilisearch/meilisearch-php/pull/22

**Pull request merged**